docs/OncoTree-Mapping-Tool.md

```diff
@@ -30,163 +30,163 @@ The list of OncoTree versions available are viewable [here](http://oncotree.mskc
- 
+|P1|S1|41||ALL -> {BLL,TLL}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P2|S2|60||BALL -> {BLL}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P3|S3|<18||TALL -> {TLL}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P4|S4|71|PTCL||<span class="oi oi-ellipses"></span>|
+|P5|S5|64|PTCL||<span class="oi oi-ellipses"></span>|
+|P6|S6|36||CHL -> {CHL}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P7|S7|63||SpCC -> ???, Oncotee code not in source oncotree version|<span class="oi oi-ellipses"></span>|
+|P8|S8|63||MCL -> {MCL}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P9|S9|73||HGNEE -> ???, Oncotee code not in source oncotree version|<span class="oi oi-ellipses"></span>|
+|P10|S10|52||ONCOTREE_CODE column blank : use a valid oncotree code or "NA"|<span class="oi oi-ellipses"></span>|
+|P11|S11|77|NA||<span class="oi oi-ellipses"></span>|
+|P12|S12|87||TNKL -> {MTNN}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P13|S13|79||HIST -> {HDCN}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P14|S14|53|CLLSLL||<span class="oi oi-ellipses"></span>|
+|P15|S15|69|CLLSLL||<span class="oi oi-ellipses"></span>|
+|P16|S16|65||LEUK -> {MNM}, more granular choices introduced|<span class="oi oi-ellipses"></span>|
+|P17|S17|66|MYCF||<span class="oi oi-ellipses"></span>|
+|P18|S18|66|RBL||<span class="oi oi-ellipses"></span>|
+
 #### Step 5
-Using values in the `ONCOTREE_CODE_OPTIONS` as a guide, manually select and place an OncoTree Code in the `ONCOTREE_CODE` column. For additional information, refer to the summary file `data_clinical_sample_remapped_summary.html`. Repeat for all rows in the output file. Several examples are shown below.  
+Using values in the `ONCOTREE_CODE_OPTIONS` as a guide, manually select and place an OncoTree Code in the `ONCOTREE_CODE` column. For additional information, refer to the summary file `data_clinical_sample_remapped_summary.html`. Repeat for all rows in the output file. Several examples are shown below.
 
- ###### Sample 1  
+###### Sample 1
 
 |SAMPLE_ID|ONCOTREE_CODE  |ONCOTREE_CODE_OPTIONS|
 |:--------|:-------------:|:----------------------:|
 |S1       |	              |ALL -> {BLL,TLL}, more granular choices introduced|
 
-Source OncoTree code `ALL` maps directly to codes `BLL` and `TLL`. Users should place either `BLL` or `TLL` in the `ONCOTRE_CODE`column. The `ONCOTREE_CODE_OPTIONS` column also notes that more granular choices were introduced; as such, users can use the summary file for additional guidance. 
+Source OncoTree code `ALL` maps directly to codes `BLL` and `TLL`. Users should place either `BLL` or `TLL` in the `ONCOTRE_CODE`column. The `ONCOTREE_CODE_OPTIONS` column also notes that more granular choices were introduced; as such, users can use the summary file for additional guidance.
```

Comment:
There is as space missing between "ONCOTREE_CODE" and "column".

Reply:
fixed